VALIDATORS_FAQ.md

```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
+* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms on the Cosmos Hub) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+* **Block rewards:** Block rewards are paid in Photons. Initial distribution of Photons will be hard spooned from Ethereum. This means Photons will be emitted 1:1 to Ether. 
 * **Transaction fees:** The Cosmos Hub maintains a whitelist of token that are accepted as fee payment. 
 
-These rewards are divided among validators' staking pools according to each validator’s weight. Then, they are divided among delegators in proportion to each delegator’s stake. Note that a commission on fee rewards for delegators is applied by the validator before they are distributed. However, no commission can be applied on block provisions (see below).
+This total revenue is divided among validators' staking pools according to each validator’s weight. Then, it is divided among delegators in proportion to each delegator’s stake. Note that a commission on delegators' revenue is applied by the validator before it is distributed. 
 
 ### What is the incentive to run a validator ?
 
-Validators earn proportionally more rewards than their delegators because of commissions on the fees that are collected. These fees can be collected not only in Atoms but in any token that has been whitelisted through governance. 
+Validators earn proportionally more revenue than their delegators because of commissions.
 
 Validators also play a major role in governance. If a delegator does not vote, it inherits the vote from its validator. This gives validators a major responsibility in the ecosystem.
 
 ### What is a validator's commission?
 
-Fee rewards received by a validator’s pool are split between the validator and its delegators. The validator can apply a commission on the part of the fees that goes to its delegators. This commission is set as a percentage. Each validator is free to set its initial commission, maximum daily commission change rate and maximum commission. The Cosmos Hub enforces the parameter that each validator sets. These parameters can only be defined when initially declaring candidacy. 
-
-Note that validators cannot set commission on block provisions. This is to prevent a continuous shift in distribution from delegators to validators which would increase the concentration of Atoms and ultimately weaken the security of the system.
+Revenue received by a validator’s pool is split between the validator and its delegators. The validator can apply a commission on the part of the revenue that goes to its delegators. This commission is set as a percentage. Each validator is free to set its initial commission, maximum daily commission change rate and maximum commission. The Cosmos Hub enforces the parameter that each validator sets. These parameters can only be defined when initially declaring candidacy. 
```

Comment:
last sentence: 
These parameters are defined when initially declaring candidacy, and may only be constrained further with time.